features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/MetaData.jsx

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import Tooltip from '@material-ui/core/Tooltip';
+import API from 'AppData/api';
+import { useAPI } from 'AppComponents/Apis/Details/components/ApiContext';
+import BusinessInformation from './BusinessInformation';
+
+/**
+ *
+ *
+ * @param {*} props
+ * @returns
+ */
+function MetaData(props) {
+    const { parentClasses } = props;
```

Comment:
Why do we need to get classes from props here ? canse we use `makeStyles` in this component ?

Reply:
Let's do a code refactoring later, I just reorganized the existing code for layout changes.